apps/web/public/fallback-MOpYCah9WS_VY3TikEHRP.js

```diff
@@ -0,0 +1,13 @@
+(() => {
+  'use strict';
+  self.fallback = async (e) => {
+    switch (e.destination) {
+      case 'document':
+        return caches.match('/_offline', { ignoreSearch: !0 });
+      case 'image':
+        return caches.match('/static/fallback.png', { ignoreSearch: !0 });
+      default:
+        return Response.error();
+    }
+  };
+})();
```

Comment:
Is this file correct?

Reply:
no, it should be inside of the .gitignore